packages/core/src/components/button/test/basic/button.e2e.ts

```diff
@@ -0,0 +1,99 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentName = 'basic';
```

Comment:
Nice! Is this a common practice, or more a personal style :)?

Reply:
Totally personal style :) 
Like for now with dropdown I do this
```
const componentTestPath = 'src/components/dropdown/test/default/index.html';
```
and then 
```
await page.goto(componentTestPath);
```

Its just so I don't have to update the `await page.goto(componentTestPath);` every time I copy a testcase :D

Reply:
We would still need to decide on what approach to take :) Or we just do whatever Martin proposed. I think it all works. I also tried `beforeEach` test but that did not work with spying on clicks and so on.

Reply:
Changed to componentTestPath becouse I think its abit more clear on what is happening and less text. :)

Reply:
I see, I think we in the team should decide on what approach we want to take. I don't know what Martin proposed. I think we are just now starting out to test so maybe we should have that talk soon. So we create some common ways of doing things. We have the "How we test" in [IXIB-book](https://scaniaazureservices-my.sharepoint.com/personal/nikola_johnny_mirkovic_scania_com/_layouts/15/Doc.aspx?sourcedoc={a2c3d5d9-d0dc-4cb9-bee5-5e8a81cedb1b}&action=edit&wd=target%28Testing.one%7C5706639b-95cd-40d8-95f6-f45fe9e25134%2FHow%20we%20write%20unit%20tests%7C7c94762d-daac-4f92-a760-89a1f3b8ddd7%2F%29&wdorigin=NavigationUrl). And then, some things will always be a personal style and that is totally fine I think.

Reply:
@timrombergjakobsson Yes it looks like I'm following the IXIB-book in this PR.
It mentions nothing about how we would like to specify playwright page.goto path so I just did what made most sense to me but we might need to spend an hour to talk about it?